libtransact/src/contract/address/mod.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod key_double_hash;
```

Comment:
feature = contract-address-double-key-hash

Reply:
these modules should probably be double_key_hash and triple_key_hash because 'double hash' sounds like hash(hash(key)).